cli/tests/testdata/cert/cafile_info.ts.out

```diff
@@ -1,6 +1,7 @@
 local: [WILDCARD]https[WILDCARD]localhost_PORT5545[WILDCARD]
 type: TypeScript
-dependencies: 8 unique (total [WILDCARD])
+dependencies: 8 unique
+size: [WILDCARD]
```

Comment:
I moved size to a separate line because it was saying "dependencies: 0 unique (total [SIZE OF ROOT MODULE])", which wasn't really correct.

Reply:
Hopefully that doesn't break anything for users :)